po/R-fr.po

```diff
@@ -0,0 +1,645 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: cleanepi 1.0.2.9000\n"
+"POT-Creation-Date: 2024-11-23 10:58+0000\n"
+"PO-Revision-Date: 2024-11-07 12:38+0000\n"
+"Last-Translator: Karim Mané\n"
+"Language-Team: none\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: check_date_sequence.R:57
+msgid ""
+"Found the following unrecognised column name{?s}: {.field "
+"{target_columns[missing_cols]}}."
+msgstr ""
+"Les colonnes suivantes ne sont pas reconnues: {.field "
+"{target_columns[missing_cols]}}."
```

Comment:
Is it possible to keep the pluralization feature from the string in English here?

Reply:
Good point - I will work on this.